docs/source/diagnosis.md

```diff
@@ -0,0 +1,85 @@
+1. [Diagnosis introduction](#diagnosis-introduction)
+2. [Supported Feature Matrix](#supported-feature-matrix)
+3. [Get started](#get-started)
+4. [Example](#example)
+
+# Diagnosis introduction
+To validate the quality of the optimized model we should run diagnostics and check its accuracy and performance.
```

Comment:
The diagnosis feature provide methods to debug the accuracy loss during quantization and profile the performance gap during benchmark. There are 2 ways ...